### DISCUSSION TITLE
Test not working for Tooltip

### DISCUSSION THREAD

**Discussion by jbertan** [saadeghi/daisyui#1369] [Q&A]
className="tooltip tooltip-primary tooltip-bottom"
          data-tip="Hello"
        

Any idea how can i test ? 
(using Jest) i couldnt reach Hello for picking this tooltip.
Help please.

**Comment by saadeghi** [comment]
I'm not sure how I can help you because I don't know what exactly you're trying to test.  
If it helps, I can tell you how it works:  
When we hover the element with `tooltip` class, the `before` pseudo element which contains the `data-tip` value becomes visible. There's also a `after` pseudo element with the same hover behavior for that small arrow.

If you tell me what you're trying to test I can help.  
Or if there's a specific issue, please provide a reproduction repo.

**Reply by jbertan** [reply]
https://github.com/jbertan/Jest-and-RTL

Hello in SummaryForm.test.tsx trying to reach  daisyUI tooltip for testing but couldnt reach any getBy method.
